### DISCUSSION TITLE
Does CARLA support 4090 or 3090 or A100 80G

### DISCUSSION THREAD

**Discussion by swsamleo** [carla-simulator/carla#5925] [Q&A] [answered]
I found black artifact in some simple test and I use A100 80G. I'd like to buy some other graphic cards but I am not sure if it supports 4090.

**Comment by germanros1987** [comment, marked as answer]
Yes to all, except for A100. A100 does not support graphics APIs (Vulkan, DirectX, etc.). You probably want to check A6000.

**Reply by swsamleo** [reply]
How about the new graphic cards such as 4090/3090?
